src/components/Timer/Timer.style.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import { theme } from "../../styles/Theme.style";
+import { LargeHeading, SmallHeading } from "../Heading/Heading.style";
+
+export const TimerWrapper = styled.div`
+  text-align: center;
+  position: relative;
+`;
+
+export const TimerRectangle = styled.img`
```

Comment:
There's no need to create 'TimerRectangle' if it doesn't add any styles. You can simply use an img element.